docs/Ways-of-Working/Communication/Documentation-Updates.md

```diff
@@ -0,0 +1,31 @@
+# Documentation Updates
+
+Documentation at AM is an ongoing project, we're currently focused on improving documentation and making it for everyone to access and discover information to help their understanding of how we work, and the platforms we work on.
```

Comment:
An ongoing ~project~ *process*, maybe?

Also - presumably missing the word "easier"?
> ...and making it **easier** for everyone...

Reply:
Thanks Fixed!